src/Analyzers/CSharp/Tests/MakeLocalFunctionStatic/MakeLocalFunctionStaticTests.cs

```diff
@@ -508,6 +508,20 @@ void B()
         }
     }
 }",
+parseOptions: CSharp8ParseOptions);
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsMakeLocalFunctionStatic)]
+        [WorkItem(53179, "https://github.com/dotnet/roslyn/issues/53179")]
```

Comment:
This issue is about about the [`CodeRefactoringProvider`](https://github.com/dotnet/roslyn/blob/main/src/Features/CSharp/Portable/MakeLocalFunctionStatic/MakeLocalFunctionStaticCodeRefactoringProvider.cs), not the [`CodeFixProvider`](https://github.com/dotnet/roslyn/blob/main/src/Features/CSharp/Portable/MakeLocalFunctionStatic/MakeLocalFunctionStaticCodeFixProvider.cs).

Might be good to add a [test](https://github.com/dotnet/roslyn/blob/main/src/EditorFeatures/CSharpTest/MakeLocalFunctionStatic/MakeLocalFunctionStaticRefactoringTests.cs) to the coderefactoring if it already works.

Reply:
Thanks, added a test. They do share the code so worked fine :-)